src/bin/bench.rs

```diff
@@ -43,6 +44,9 @@ Options:
     --manifest-path PATH     Path to the manifest to build benchmarks for
     -v, --verbose            Use verbose output
     -q, --quiet              No output printed to stdout
+    --color WHEN             Configure coloring of output: auto = colorize, if
+                             output goes to a tty (default); always = always
+                             colorize output; never = never colorize output
```

Comment:
Can this description be a little more concise? It'd be nice to keep it on one line